app/domain/authentication/authn_jwt/authentication_parameters.rb

```diff
@@ -2,15 +2,14 @@ module Authentication
   module AuthnJwt
     # Data class to store data regarding jwt token that is needed during the jwt authentication process
     class AuthenticationParameters
-      attr_accessor :jwt_identity, :decoded_token, :jwt_token
-      attr_reader :authenticator_name, :service_id, :account, :username, :credentials, :client_ip, :request
+      attr_accessor :jwt_identity, :decoded_token, :jwt_token, :credentials
```

Comment:
is ti possible to not have accessors? can (at least some of them) be initialized in the init method?